Makefile

```
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CSI_VERSION ?= main
+GCSFUSE_VERSION ?= $(shell tr -dc A-Za-z0-9 </dev/urandom | head -c 6)
```

Checking if we can do something like this:
```suggestion
GCSFUSE_VERSION ?= $(shell git rev-parse --short HEAD)
```